File: C9_Server_Rest_&_WEB/clases/__doctrine/src/Cita.php
<?php
/**
 * @Entity @Table(name="cita")
 **/
class Cita{
    /**
     * @Id
     * @Column(type="integer") @GeneratedValue
     */
    protected $id;
    
    /**
     * @Column(type="string", length=10, unique=false, nullable=false)
     */
    protected $seguridadSocial;
    
    /**
     * @Column(type="string", length=25, unique=false, nullable=false)
     */
    protected $type;
    
    /**
     * @Column(type="string", length=200, unique=false, nullable=false)
     */
    protected $reason;
    
    /**
     * @Column(type="string", length=9, unique=false, nullable=false)
     */
    protected $telephone;
    
    /**
     * @Column(type="date",unique=false,nullable=false)
     */
    protected $fecha;
    //Methods Getter & Setter
    
    public function getId(){
        return $this->id;
    }
    
    public function setId($id){
        $this->id = $id;
    }
    
    public function getSeguridadSocial(){
        return $this->seguridadSocial;
    }
    
    public function setSeguridadSocial($number){
        $this->seguridadSocial = $number;
    }
    
    public function getName(){
        return $this->name;
    }
    
    public function setName($name){
        $this->name = $name;
    }
    
    public function getType(){
        return $this->type;
    }
    
    public function setType($type){
        $this->type = $type;
    }
    
    public function getReason(){
        return $this->reason;
    }
    
    public function setReason($reason){
        $this->reason = $reason;
    }
    
    public function getTelephone(){
        return $this->telephone;
    }
    
    public function setTelephone($telephone){
        $this->telephone = $telephone;
    }
    
    public function getFecha(){
        return $this->fecha;
    }
    
    public function setFecha($fecha){
        $this->fecha = $fecha;
    }
    
    public function getArray(){
        return array(
                "id" => $this->id,
                "seguridadSocial" => $this->seguridadSocial,
                "type" => $this->type,
                "reason" => $this->reason,
                "telephone" => $this->telephone,
                "fecha" => $this->fecha
            );
    }
    
    function __toString(){
        return 'Cita: ' . $this->getId() . ' ' . $this->getSeguridadSocial() . ' ' . 
        $this->getName() . ' ' . $this->getReason() . ' ' . $this->getType() . ' ' . 
        $this->getTelephone();
    }
}

File: consultasMedicasApp/app/src/main/java/com/example/josea/consultasmedicas/pojo/Cita.java
package com.example.josea.consultasmedicas.pojo;

import android.os.Parcel;
import android.os.Parcelable;

import java.util.Date;

/**
 * Created by josea on 23/05/2017.
 */

public class Cita implements Parcelable{

    private int id;
    private String seguridadSocial;
    private String type;
    private String reason;
    private String telephone;
    private Date fecha;

    public Cita(int id, String seguridadSocial, String type, String reason, String telephone, Date fecha) {
        this.id = id;
        this.seguridadSocial = seguridadSocial;
        this.type = type;
        this.reason = reason;
        this.telephone = telephone;
        this.fecha = fecha;
    }

    public Cita(){}

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getSeguridadSocial() {
        return seguridadSocial;
    }

    public void setSeguridadSocial(String seguridadSocial) {
        this.seguridadSocial = seguridadSocial;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getReason() {
        return reason;
    }

    public void setReason(String reason) {
        this.reason = reason;
    }

    public String getTelephone() {
        return telephone;
    }

    public void setTelephone(String telephone) {
        this.telephone = telephone;
    }

    public Date getFecha() {
        return fecha;
    }

    public void setFecha(Date fecha) {
        this.fecha = fecha;
    }

    @Override
    public String toString() {
        return "Cita{" +
                "id=" + id +
                ", seguridadSocial='" + seguridadSocial + '\'' +
                ", type='" + type + '\'' +
                ", reason='" + reason + '\'' +
                ", telephone='" + telephone + '\'' +
                ", fecha=" + fecha +
                '}';
    }

    @Override
    public int describeContents() {
        return 0;
    }

    @Override
    public void writeToParcel(Parcel dest, int flags) {
        dest.writeInt(id);
        dest.writeString(seguridadSocial);
        dest.writeString(type);
        dest.writeString(reason);
        dest.writeString(telephone);
        dest.writeString(fecha.toString());
    }
    protected Cita(Parcel in) {
        id = in.readInt();
        seguridadSocial = in.readString();
        type = in.readString();
        reason = in.readString();
        telephone = in.readString();
        fecha = new Date();

    }

    public static final Creator<Cita> CREATOR = new Creator<Cita>() {
        @Override
        public Cita createFromParcel(Parcel in) {
            return new Cita(in);
        }

        @Override
        public Cita[] newArray(int size) {
            return new Cita[size];
        }
    };
}
